architect.yml

```diff
@@ -0,0 +1,40 @@
+name: flask
+description: A starter component that uses Flask framework and SQLite database.
+
+keywords:
+  - Flask
+  - SQLite
+  - Python
+  - HTML
+  - Gunicorn
+
+parameters:
+  flask_env:
+    description: Environment in which Flask application runs
+    default: production
+  gunicorn_workers:
+    description: Number of Gunicorn worker processes
+    default: 4
+
+services:
+  app:
+    build:
+      context: ./
+    interfaces:
+      main: &app-port 5000
+    environment:
+      FLASK_ENV: ${{ parameters.flask_env }}
+      FLASK_PORT: *app-port
+    command:
+      - sh
+      - -c
+      - |
+        gunicorn -w ${{ parameters.gunicorn_workers }} -b 0.0.0.0:5000 "flaskr:create_app()"
```

Comment:
I think we should set the `5000` here to `${{ services.app.interfaces.main.url }}`